inequality/atkinson.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+__all__ = ["Atkinson"]
+
+
+def _atkinson(y, epsilon):
```

Comment:
this has a really similar API to segregation, where the function is private and the statistic stored internally as an attributed on the class. As we've talking about in segregation and esda, it would probably be nice to expose both function and OO api's (i.e. by tweaking the structure a bit and making `_atkinson` public).

Just dropping this note here as something for us to consider as we work on unifying standards across the meta